pypeit/reduce.py

```diff
@@ -287,6 +289,48 @@ def local_skysub_extract(self, sciimg, sciivar, tilts, waveimg, global_sky, rn2i
 
         return None, None, None, None, None
 
+
+    def flexure_correct(self, sobjs, basename):
+        """ Correct for flexure
+
+        Args:
+            sobjs: SpecObjs object
+            maskslits: ndarray
+
+        Returns:
+            Spectra are modified in place (wavelengths are shifted)
+        """
+
+        if self.flex_par['method'] != 'skip':
+            flex_list = wave.flexure_obj(sobjs, self.maskslits, self.flex_par['method'],
+                                         self.flex_par['spectrum'],
+                                         mxshft=self.flex_par['maxshift'])
+            # QA
+            wave.flexure_qa(sobjs, self.maskslits, basename, self.det, flex_list,out_dir=self.par['rdx']['redux_path'])
+        else:
+            msgs.info('Skipping flexure correction.')
+
+
+    def helio_correct(self, sobjs, radec, obstime):
```

Comment:
need doc string

Reply:
You wrote this function not me. Practice what you preach!! I don't think it is worth updating at this point since we need to completely rethink how we are doing heliocentric corrections and flexure corrections.